images/linux/scripts/installers/graalvm.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+#Install GraalVM
+tmp_graalvm="/tmp/graalvm"
+mkdir -p "$tmp_graalvm"
+url=$(curl -s https://api.github.com/repos/graalvm/graalvm-ce-builds/releases/latest | jq -r '.assets[].browser_download_url | select(contains("graalvm-ce-java11-linux-amd64"))')
+download_with_retries "$url" "$tmp_graalvm/graalvm-archive.tar.gz"
+mkdir /usr/local/graalvm
+tar -xzf "$tmp_graalvm/graalvm-archive.tar.gz" -C /usr/local/graalvm
+setEtcEnvironmentVariable "GRAALVM_BIN" /usr/local/graalvm/graal*/bin
+
+#Install Native Image
+$GRAALVM_BIN/gu install native-image
+
+# Run tests to determine that the software installed as expected
+if [[ $($GRAALVM_BIN/java --version) != *"GraalVM"* ]]; then
+    echo "GraalVM was not installed"
+    exit 1
+fi
+
+if ! command -v $GRAALVM_BIN/native-image; then
```

Comment:
I think we should revisit variable name.
What if we add GraalVM for Java 8 in future? Please sync with @dmitry-shibanov on it

Reply:
Now we have `GRAALVM_11_BIN`  and `JAVA_HOME_GRAALVM_11` variables for GraalVM on Java 11